packages/react/src/useField.js

```diff
@@ -234,7 +234,14 @@ const useField = ({
     isChecked: (
       isCheckboxElement
       ? (
-        Boolean(
+        isMultipleElement
+        ? (
+          value
+          .includes(
+            inputValue
+          )
+        )
+        : Boolean(
```

Comment:
👍 Very simple workaround in this particular use case, but doesn't scale if we had more input types.

Since input types are defined by HTML, I think we're fine. `type="checkbox"` and `multiple` are the only ones that exist today.